Make Email tests not fail when Docker is not running

**What needs to be done?**

The email tests in the `engine` module rely on Testcontainers. Without a running Docker environment they are failing.

```
[ERROR] Errors: 
[ERROR]   AsyncEmailTaskTest.testSimpleTextMail » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   AsyncEmailTaskTest.testSimpleTextMailSendTask » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailSendTaskTest.testCcAndBcc » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailSendTaskTest.testHtmlMail » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailSendTaskTest.testSendEmail » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailSendTaskTest.testSimpleTextMail » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailSendTaskTest.testSimpleTextMailMultipleRecipients » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailSendTaskTest.testTextMailExpressions » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailServiceTaskTest.testCcAndBcc » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailServiceTaskTest.testHtmlMail » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailServiceTaskTest.testSendEmail » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailServiceTaskTest.testSimpleTextMail » IllegalState Could not find a valid Docker environment. Please see logs and check configuration
[ERROR]   EmailServiceTaskTest.testSimpleTextMailMultipleRecipients » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[ERROR]   EmailServiceTaskTest.testTextMailExpressions » IllegalState Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
[INFO] 
[ERROR] Tests run: 16060, Failures: 0, Errors: 14, Skipped: 85
[INFO] 

```

These tests should be skipped when Docker is not running on the machine.

--- a/engine/src/test/java/org/operaton/bpm/engine/test/bpmn/mail/EmailTestCase.java
+++ b/engine/src/test/java/org/operaton/bpm/engine/test/bpmn/mail/EmailTestCase.java
@@ -49,7 +49,7 @@ import static org.assertj.core.api.Assertions.fail;
 /**
  * @author Joram Barrez
  */
-@Testcontainers
+@Testcontainers(disabledWithoutDocker = true)
 public abstract class EmailTestCase {
 
   @RegisterExtension

